draft-ietf-quic-tls.md

```diff
@@ -2337,18 +2352,19 @@ distinguishing advantage between a real and random AEAD algorithm gained by an
 attacker is:
 
 ~~~
-2 * (q * l)^2 / 2^128
+2 * (q * l)^2 / 2^n
```

Comment:
I forgot to call this out.  @chris-wood, can you check that this is right here?  I think that this comes from the `\sigma*B/2^n` term in Bad3 in [the paper](https://eprint.iacr.org/2018/993.pdf), but I want to make sure that my understanding is correct in that we are treating `\sigma*B` as `\sigma^2` in the single-user case.